applications/plugins/MultiThreading/examples/ParallelCollisionDetection.scn

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<Node name="root" gravity="0 -9.81 0">
+    <RequiredPlugin name="SofaGeneralDeformable"/>
+    <RequiredPlugin name="SofaMiscCollision"/>
+    <RequiredPlugin name="SofaImplicitOdeSolver"/>
+    <RequiredPlugin name="SofaMeshCollision"/>
+    <RequiredPlugin name="SofaTopologyMapping"/>
+    <RequiredPlugin name="SofaLoader"/>
+    <RequiredPlugin name="MultiThreading"/>
+
+    <VisualStyle displayFlags="showCollisionModels"/>
+
+    <DefaultPipeline/>
+
+    <!--
+    Parallel collision detection.
+    To compare to the single thread collision detection, replace the two following components with:
+    <BruteForceBroadPhase/>
+    <BVHNarrowPhase/>
+    -->
+    <ParallelBruteForceBroadPhase/>
+    <ParallelBVHNarrowPhase/>
+
+    <NewProximityIntersection name="Proximity" alarmDistance="2" contactDistance="0.7"/>
```

Comment:
MinProximityIntersection is faster than NewProximityIntersection. Since it is faster, the difference between the single thread and the multi thread scene is reduced.
LocalMinDistance gives incorrect results.